### ISSUE TITLE
NullPointerException if project.clj specifies unversioned dependency

### ISSUE THREAD

**Issue by mfikes** [cursiveclojure/cursive#347]
Set up a project that specifies a dependency, but fails to specify the version for that dependency. For example, `project.clj`, specifying `criterium` but without a version in its vector:

```clj
(defproject foo "0.1.0-SNAPSHOT"
   :dependencies [[org.clojure/clojure "1.5.1"]
                  [criterium]])
```

When running the REPL in IntelliJ, or simply loading the project,  this results in an NPE:

```java
java.lang.NullPointerException
	at cursive.leiningen.project.LeiningenProjectsManager.getFilteredDependencies(LeiningenProjectsManager.java:415)
	at cursive.leiningen.project.LeiningenProjectsManager.access$800(LeiningenProjectsManager.java:32)
	at cursive.leiningen.project.LeiningenProjectsManager$3.run(LeiningenProjectsManager.java:267)
	at cursive.leiningen.LeiningenUtil$4$1.run(LeiningenUtil.java:101)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl$TaskRunnable.run(ProgressManagerImpl.java:464)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl$2.run(ProgressManagerImpl.java:178)
	at com.intellij.openapi.progress.ProgressManager.executeProcessUnderProgress(ProgressManager.java:209)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:212)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.runProcess(ProgressManagerImpl.java:171)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl$8.run(ProgressManagerImpl.java:373)
	at com.intellij.openapi.application.impl.ApplicationImpl$8.run(ApplicationImpl.java:419)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
	at com.intellij.openapi.application.impl.ApplicationImpl$1$1.run(ApplicationImpl.java:149)
```

**Comment by danielcompton** [comment]
I can confirm this still happens in 0.1.58.

**Comment by cursiveclojure** [comment]
This is fixed in 0.1.64.
